src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -21,18 +21,30 @@
     but the logic at runtime will never try to use the Registry if it's not available
     -->
     <PackageReference Include="Microsoft.Win32.Registry" Version="4.6.0" PrivateAssets="all" />
-  </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
     <!-- Instrumented libraries -->
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.0" PrivateAssets="All" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <!-- MessagePack -->
+  <PropertyGroup>
+    <!-- force MessagePack 1.9 when targeting net45, default to MessagePack 1.9 otherwise -->
+    <DefineConstants Condition="'$(MessagePackVersion)' == '1.9' OR '$(MessagePackVersion)' == '' OR '$(TargetFramework)' == 'net45'">$(DefineConstants);MESSAGEPACK_1_9</DefineConstants>
```

Comment:
These condition are here to allow me to run tests comparing 1.9 vs 2.1. I will probably remove them and keep only 1.9 before merging this PR.

Reply:
Removed.